src/System.Management.Automation/engine/Interop/Windows/VariantClear.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+#if !UNIX
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Windows
```

Comment:
Using nested classes doesn't feel right. Why not have two classes `WindowsInterop` and `UnixInterop`?
Also, no namespace?

Reply:
I use a pattern which .Net team uses. I hope they didn't take the worst path.:smile:
At least it is more convenient to use in the editor (IntelliSense) and it doesn't make the code generation any worse.

Reply:
If there is no particular benefit for nested classes and non-namespace, then I think we should use separate classes and with namespace.

Reply:
What you're talking about now looks like `System.Management.Automation.Platform.Unix.NativeMethods.CreateHardLink()` is terrible compared to `Interop.Windows.CreateHardLink()`.
I used the pattern that was the shortest and most convenient.
In fact, there are now at least 4 patterns used in PowerShell code!
So I used what the .Net team uses. The intention is to translate first Windows and then Unix code to this pattern.

Reply:
I'm not asking to use the original name. It can be something like below, so when using, it's `WindowsInterop.CreateHardLink()`
```
namespace System.Management.Automation
{
    internal static partial class WindowsInterop
    {
        ...
    }

    internal static partial class UnixInterop
    {
        ...
    }
}
```

Reply:
I could put all in the namespace (even though it looks like a formality) in batch after all PRs will be merged.
Names of the form Interop.Windows are, I believe, more convenient and natural. For example, it would be easier for anyone to discover new names if Interop.Android or Interop.FreeBSD appeared.

Reply:
I'm ok following .NET team's design here.

Reply:
Well, the boss said so😄 I will resolve this comment.